### PR TITLE
Fetch Json

### PR DESCRIPTION
**Pull request by pohlinwei**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -131,9 +131,9 @@ const fetchAndShowResponse = () => {
   } catch (err) {
     console.log(error);
   }
-  fetch('/data').then(response => response.text()).then(comment => {
-    commentPlaceholder.innerHTML = comment;
-  })
+  fetch('/data').then(response => response.text()).then(responseStrings => {
```
Comment (fanwgwg) [inline portfolio/src/main/webapp/script.js]: On the client side how do you know that the response is a list of string?

**Review by fanwgwg — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (pohlinwei) [reply portfolio/src/main/webapp/script.js]: Oh. HAHAH. Sorry, my bad. Will change it 😂😂

Comment (pohlinwei) [reply portfolio/src/main/webapp/script.js]: Updated! Is this okay now?

Comment (fanwgwg) [reply portfolio/src/main/webapp/script.js]: Oh I don't actually mean that. My point is that the API is `/data`, without explicitly explaining what data it is and in the response you're expecting `responseStrings`, which seems to be a lack of contract. I think you can have more detailed javadoc on the servelet to define the contract between them:)

Comment (pohlinwei) [reply portfolio/src/main/webapp/script.js]: Ahh... Great point! I have updated it. Let me know if this is better.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -14,19 +14,28 @@
...
+import java.util.List;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns a sample list of strings. */
 @WebServlet("/data")
```
Comment (fanwgwg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Do you mind also updating the api to `/sample_strings` etc?

**Review by fanwgwg — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (pohlinwei) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Of course! I like the suggestion too. Thanks! :D

**Pull request merged**